tutor/resources/styles/book-content/note.less

```diff
@@ -162,7 +162,7 @@
   }
 
   // .example, check-understanding content currently lacks a data-label attribute
-  > .example,
+  .book-content > .example,
   section > .example {
     .tutor-book-before-manual-label("Worked Example");
   }
```

Comment:
i think too large for now...should be safe, but i think it'd be better to do it with the upgrades